django_project/feti/serializers/step_detail_serializer.py

```diff
@@ -11,11 +11,12 @@ class Meta:
 
     def to_representation(self, instance):
         res = super(StepDetailSerializer, self).to_representation(instance)
-        if not instance.course.all():
+        courses = instance.course.all()
+        if not courses:
```

Comment:
instead of using `if not courses`, maybe use `if courses`?
because it return same which is res